app/models/rental.rb

```diff
@@ -0,0 +1,49 @@
+class Rental < ApplicationRecord
+  belongs_to :customer
+  belongs_to :movie
+
+  def self.check_dependencies(customer, movie)
+    if customer.nil?
+      return {
+        customer_id: ["No such customer"]
```

Comment:
Since the customer and movie are about to be associated with an instance of `Rental`, it might make sense to make this an instance method.